h/templates/layouts/base.html.jinja2

```diff
@@ -29,6 +29,35 @@
     {% endfor -%}
     {% block styles %}{% endblock %}
 
+    <link rel="apple-touch-icon" sizes="57x57"
```

Comment:
Just the 152x152 (or 144x144) icons should be sufficient - see https://mathiasbynens.be/notes/touch-icons and https://github.com/h5bp/html5-boilerplate/issues/1367

The tiny downside is that older devices download the larger image and then rescale it.